backend/acl/sink.go

```diff
@@ -0,0 +1,34 @@
+package acl
+
+import (
+	"context"
+
+	"github.com/uw-labs/proximo"
+	"github.com/uw-labs/proximo/proto"
+	"github.com/uw-labs/substrate"
+)
+
+type AsyncSinkFactory struct {
+	cfg  Config
+	sink proximo.AsyncSinkFactory
+}
+
+func ProximoACLSinkFactory(cfg Config, factory proximo.AsyncSinkFactory) (proximo.AsyncSinkFactory, error) {
+	return &AsyncSinkFactory{
+		cfg:  cfg,
+		sink: factory,
+	}, nil
+}
```

Comment:
I would really prefer to have the setup like this instead, as that's how it's setup for nats streaming (https://github.com/uw-labs/proximo/blob/master/backend/natsstreaming/natsstreaming.go#L12-L20) and kafka. Also the error is completely unnecessary in the return of the constructor. Do you have anything against this approach?
```suggestion
type AsyncSinkFactory struct {
	Config  Config
	Next proximo.AsyncSinkFactory
}
```